index.html

```diff
@@ -662,7 +662,45 @@ <h2>Benefits of JSON-LD and LD-Proofs</h2>
 the cost of implementation complexity.
       </dd>
     </dl>
-
+    <p>
+The Verifiable Credentials Data Model is designed to be compatible with a
+variety of existing and emerging digital proof formats. Each
+proof format has benefits and drawbacks. Many proof formats can only reveal all
+attribute values in a verifiable credential.
```

Comment:
It took some work to extract the meaning of this sentence --
> Many proof formats can only reveal all attribute values in a verifiable credential.

Perhaps something like this?
> For instance, many proof formats cannot reveal selected attribute values in a verifiable credential; they can only reveal all (or none).

Reply:
I like your suggestion. I will update the text.

Reply:
Added @TallTed suggestion.